### DISCUSSION TITLE
Relations: The distinctions between Publishes and Sets unclear

### DISCUSSION THREAD

**Discussion by MikeHypercube** [edmcouncil/fibo#1354] [Indices and Indicators (IND)]
The distinctions between Publishes and Sets, are not very clear, and re-use the same Relations property. 

MB response: We made these separate sub-properties in order to support the difference in meaning, under one more general relationship. So they are intended to reuse the same property and they are intended to represent different meaningful refinements of the meaning of that property. 

Please propose improvements.

**Comment by trypuz** [comment]
@MikeHypercube, could you provide IRIs of the properties we are discussing in this issue, please. Has anything changed in the subject, since 2014 when this task was created?

**Comment by rivettp** [comment]
We have https://spec.edmcouncil.org/fibo/ontology/IND/InterestRates/InterestRatePublishers/sets with definition "a predicate indicating that some party places something in some relationship to something or someone else" and example "The interest rate authority (such as a central bank or monetary authority or a panel working behalf of such) determines and sets the reference rate which is in force at a given time."
and https://spec.edmcouncil.org/fibo/ontology/BE/FunctionalEntities/Publishers/publishes with definition "prepares and issues material for public consumption".

Then we have the restriction on https://spec.edmcouncil.org/fibo/ontology/IND/InterestRates/InterestRatePublishers/MarketDataProvider on property _publishes_ at least one _MarketRate_
but https://spec.edmcouncil.org/fibo/ontology/IND/InterestRates/InterestRatePublishers/InterestRateAuthority (a subclass of FinancialInformationProvider) has a restriction using property _sets_ on _ReferenceInterestRate_.
It seems that (a more specific subproperty) of _sets_ should also be a subproperty of _publishes_ (since when would anyone set a rate without publishing it?) or existing restrictions on _sets_ should be accompanied by restrictions on _publishes_.
